CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
 Contributing
 ------------
 
-Symfony CMF is an open source, community-driven project. We follow the same
-guidelines as core Symfony. If you'd like to contribute, please read the
+Symfony2 CMF is an open source, community-driven project. We follow the same
```

Comment:
this should say "Symfony CMF", not Symfony2 CMF. the official name of symfony moved from Symfony2 to Symfony, as symfony 3 and 4 are still symfony.

Reply:
This should be updated in mostly all packages then.